{{cookiecutter.repo_name}}/docs/source/_static/logo/placeholder_favicon.svg

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Maybe I don't understand the docs in your latest PR on the theme repo - isn't this already available upstream or is this some other thing completely?

Reply:
Or is the intent to have the placeholder there so that folks know what to replace?

Reply:
Yeah the placeholder is there because otherwise it may not be totally clear what the relative path is or what the correct keyword would be. Also having the folder there standardises naming